docs/source/array-overlap.rst

```diff
@@ -0,0 +1,181 @@
+Overlapping Computations
+========================
+
+Some array operations require communication of borders between neighboring
+blocks.  Example operations include the following:
+
+*  Convolve a filter across an image
+*  Sliding sum/mean/max, ...
+*  Search for image motifs like a Gaussian blob that might span the border of a
+   block
+*  Evaluate a partial derivative
+*  Play the game of Life_
+
+Dask array supports these operations by creating a new array where each
+block is slightly expanded by the borders of its neighbors.  This costs an
+excess copy and the communication of many small chunks but allows localized
+functions to evaluate in an embarrassingly parallel manner.
+
+The main API for these computations is the ``map_overlap`` method defined
+below:
+
+.. currentmodule:: dask.array
+
+.. autosummary::
+   map_overlap
+
+.. autofunction:: map_overlap
+
+
+Explanation
+-----------
+
+Consider two neighboring blocks in a Dask array.
+
+.. image:: images/unghosted-neighbors.png
```

Comment:
Kind of nitpicky, but should we rename the images as well?

Reply:
Worth noting there are SVGs like this as well that may need the same renaming. SVGs contain a `docname` parameter internally (why?) that includes the name too, which would need renaming as well.